### PR TITLE
remove releases from artifact downloads

### PR DESCRIPTION
**Pull request by stevekuznetsov**
@danmcp PTAL

### CODE REVIEW

**Comment by danmcp** [comment]
@stevekuznetsov Don't we still need this for devenv_ami?

**Comment by stevekuznetsov** [comment]
@danmcp oh, right, that uses the same script. how can we tell at runtime which job we're in?

**Comment by danmcp** [comment]
@stevekuznetsov I think you need to add an option

**Comment by stevekuznetsov** [comment]
To `test-origin`?

**Comment by danmcp** [comment]
@stevekuznetsov yes

**Comment by stevekuznetsov** [comment]
@danmcp ready for a look

File: lib/vagrant-openshift/command/download_artifacts_origin.rb [hunk 1]
```diff
@@ -31,6 +31,11 @@ def execute
           opts = OptionParser.new do |o|
             o.banner = "Usage: vagrant download-artifacts-origin [machine-name]"
             o.separator ""
+
+            o.on("", "--keep-release", String, "Download release binaries") do |f|
```
Comment (danmcp) [inline lib/vagrant-openshift/command/download_artifacts_origin.rb]: how about --include-release-artifacts

File: lib/vagrant-openshift/command/test_origin.rb [hunk 2]
```diff
@@ -59,6 +59,10 @@ def execute
               options[:download] = true
             end
 
+            o.on("", "--keep-release", String, "Download release binaries") do |f|
+              options[:download_release] = true
```
Comment (danmcp) [inline lib/vagrant-openshift/command/test_origin.rb]: how about --download-release-artifacts

**Comment by stevekuznetsov** [comment]
@danmcp updated flag names

File: lib/vagrant-openshift/action/download_artifacts_origin.rb [hunk 3]
```diff
@@ -21,9 +21,10 @@ module Action
       class DownloadArtifactsOrigin
         include CommandHelper
 
-        def initialize(app, env)
+        def initialize(app, env, options)
           @app = app
           @env = env
+          @options = options.clone
```
Comment (danmcp) [inline lib/vagrant-openshift/action/download_artifacts_origin.rb]: why clone?

Comment (stevekuznetsov) [inline lib/vagrant-openshift/action/download_artifacts_origin.rb]: copy-pasted that from the way that it was done in `test_origin`

Comment (danmcp) [inline lib/vagrant-openshift/action/download_artifacts_origin.rb]: I don't think it's needed here but it's harmless I guess.

**Pull request merged**